sn_faucet/src/faucet_server.rs

```diff
@@ -203,6 +216,19 @@ async fn respond_to_gift_request(
     key: String,
     semaphore: Arc<Semaphore>,
 ) -> std::result::Result<impl Reply, std::convert::Infallible> {
+    let faucet_root = get_faucet_data_dir();
```

Comment:
maybe this block of `loading wallet` code can be a separate private helper function?
seems appeared duplicatedly couple of times

Reply:
it's only twice in that file, not sure it's necessary atm.